01-data/01-lesson/01-01-lesson.Rmd

```diff
@@ -410,11 +410,12 @@ Instead we can create a new variable, named `read_cat`, with the `mutate()`
 function and the helpful `if_else()` function. 
 
 ```{r hsb2-mutate-read-cat, exercise=TRUE}
-hsb2 <- hsb2 %>%
+(hsb2 <- hsb2 %>%
   mutate(read_cat = if_else(read < avg_read,
                             "below average",
                             "at or above average")
          )
+)
```

Comment:
```suggestion

hsb2
```

Reply:
I think I prefer to just type out the dataset name to see the effect since the parentheses approach isn't covered well elsewhere in the tutorials.